guides/common/modules/con_managing-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,32 @@
+[id="managing_multiple_content_view_environments_{context}""]
+= Managing multiple content view environments
```

Comment:
Gerunds are used for procedures. This is conceptual information. Better to change it to something like "About multiple content view environments" or similar.